manifests/base/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ kind: Kustomization
 images:
 - name: quay.io/argoproj/argocd
   newName: quay.io/argoproj/argocd
-  newTag: v2.12.8
+  newTag: latest
```

Comment:
Given that we are merging this PR in release-2.12, we would not want to change this tag to `latest`. The failing CI for generated code is a known issue for this scenario.

Reply:
reverted